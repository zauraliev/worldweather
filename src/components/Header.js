import React from "react";

const Header = () => (
  <div className="header">
    <h1 className="font-weight-normal">
      World<span className="font-weight-light">Weather</span>
    </h1>
    <small>Weather by City and Country Across the World</small>
  </div>
);

export default Header;
