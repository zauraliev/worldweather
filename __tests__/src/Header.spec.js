import React from "react";
import { shallow } from "enzyme";
import renderer from "react-test-renderer";
import Header from "../../src/components/Header";

describe("Header", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<Header />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
