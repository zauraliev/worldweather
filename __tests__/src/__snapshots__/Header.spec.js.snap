// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<div
  className="header"
>
  <h1
    className="font-weight-normal"
  >
    World
    <span
      className="font-weight-light"
    >
      Weather
    </span>
  </h1>
  <small>
    Weather by City and Country Across the World
  </small>
</div>
`;
